apps/templates/filters.py

```diff
@@ -63,9 +63,9 @@ def first_paragraph_filter(input_string):
     return ""
 
 
-def add_timedelta(date, minutes):
+def add_timedelta(date, **kwargs):
     try:
-        return date + timedelta(minutes=minutes)
+        return date + timedelta(minutes=kwargs.get("minutes"))
```

Comment:
it should be rather `timedelta(**kwargs)` here so there can be hours/seconds/whatever

Reply:
yep, I will do the changes.